specification/cognitiveservices/data-plane/Language/stable/2022-05-01/customtextauthoring.json

```diff
@@ -0,0 +1,3000 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Language - Custom Text Authoring APIs (2022-05-01)",
+    "version": "2022-05-01",
+    "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms. The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/overview</a>."
+  },
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "description": "A subscription key for a Language service resource.",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/language",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "common.json#/parameters/Endpoint"
+      }
+    ]
+  },
+  "paths": {
+    "/authoring/analyze-text/projects": {
```

Comment:
Same comment as for the other swagger about /authoring and file name, though I see from https://github.com/Azure/azure-rest-api-specs/blob/4caf2df6566101e2a17a85b7afd856aa534c1429/specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/textanalytics.json#L31 this file should probably be called _textanalytics-authoring.json_.

Reply:
Replied to the comment above with the reason for the word `authoring` in the path.